src/main/java/com/limechain/client/FullNode.java

```diff
@@ -7,39 +7,26 @@
 import com.limechain.storage.KVRepository;
 import com.limechain.storage.block.BlockStateHelper;
 import com.limechain.sync.warpsync.WarpSyncMachine;
-import com.limechain.trie.structure.BranchNodeHandle;
-import com.limechain.trie.structure.NodeHandle;
-import com.limechain.trie.structure.StorageNodeHandle;
-import com.limechain.trie.structure.TrieNodeIndex;
-import com.limechain.trie.structure.TrieStructure;
-import com.limechain.trie.structure.Vacant;
-import com.limechain.trie.structure.decoded.node.DecodedNode;
-import com.limechain.trie.structure.decoded.node.StorageValue;
-import com.limechain.trie.structure.nibble.BytesToNibbles;
+import com.limechain.trie.structure.InsertTrieBuilder;
+import com.limechain.trie.structure.TrieBuildException;
 import com.limechain.trie.structure.nibble.Nibble;
-import com.limechain.trie.structure.nibble.Nibbles;
 import com.limechain.trie.structure.node.InsertTrieNode;
 import com.limechain.trie.structure.node.NodeChildData;
 import com.limechain.trie.structure.node.TrieNodeData;
-import com.limechain.utils.HashUtils;
 import lombok.SneakyThrows;
 import lombok.extern.java.Log;
-import org.apache.commons.lang3.ArrayUtils;
-import org.javatuples.Pair;
 
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
 import java.util.logging.Level;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
 @Log
 public class FullNode implements HostNode {
+    private static final String TRIE_NODE_PREFIX = "tn:";
+    private static final String TRIE_NODE_STORAGE_PREFIX = "tns:";
+    private static final String TRIE_NODE_CHILD_PREFIX = "tnc:";
+
```

Comment:
(no action required) nit: there are many different prefix conventions throught the code. We should probably settle for one at some point.
<img width="804" alt="image" src="https://github.com/LimeChain/Fruzhin/assets/29047760/0f5f27a2-7ade-4908-81ad-563acce86017">

Reply:
I think we can even move them to an enum or class and access them from there. I opened a separate issue for that: #278